### PR TITLE
Check for an operator config change before updating

### PR DESCRIPTION
**Pull request by mrogers950**
This should stop us from continually updating the operator config and doing a handful of GETs for operator resources each time.
@openshift/sig-auth

### CODE REVIEW

File: pkg/operator/operator.go [hunk 1]
```diff
@@ -183,8 +186,10 @@ func (c serviceCertSignerOperator) Sync(obj metav1.Object) error {
 		operatorConfig.Status.ObservedGeneration = operatorConfig.ObjectMeta.Generation
 	}
 
-	if _, err := c.operatorConfigClient.ServiceCertSignerOperatorConfigs().UpdateStatus(operatorConfig); err != nil {
-		errors = append(errors, err)
+	if !reflect.DeepEqual(operatorConfigOriginal, operatorConfig) {
```
Comment (ericavonb) [inline pkg/operator/operator.go]: Can use https://godoc.org/k8s.io/apimachinery/pkg/api/equality

**Comment by mrogers950** [comment]
@ericavonb updated

**Comment by ericavonb** [comment]
This is a good improvement. 👍
/lgtm

**Pull request merged**